.travis.yml

```diff
@@ -1,6 +1,5 @@
 
 # .travis.yaml
-#
```

Comment:
Can you please revert this change to keep the file consistent with others?

Reply:
Thanks for the review. I would do that.